components/elements/digit-switch/DigitSwitch.element.jsx

```diff
@@ -17,6 +18,7 @@ const DigitSwitch = ({
 }) => (
   <DigitControlLabelWithError
     error={error}
+    label={error != null && error ? errorMessage : label}
```

Comment:
null check is unnecessary, `error ? errorMessage : label` will behave exactly the same.